### PR TITLE
Add second blog post

### PR DESCRIPTION
**Pull request by Mihnea0Firoiu**
Add second blog post for GSoC'24: Linux x86 Boot Protocol Support

### CODE REVIEW

**Comment by StefanJum** [comment]
This looks fine, I guess the contents might change after the call with @mogasergiu we have planned for today

File: content/blog/2024-07-10-unikraft-gsoc-lxboot_x86.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+* 0x202: HEADER
+* 0x206: VERSION
+* 0x211: LOADFLAGS
+* 0x236: XLOADFLAGS
+* 0x22c: INITRD_ADDR_MAX
+* 0x1f1: SETUP_SECTS
+
+And 5 are written by QEMU:
+
+* 0x228: CMD_LINE_PTR
```
Comment (mogasergiu) [inline content/blog/2024-07-10-unikraft-gsoc-lxboot_x86.mdx:51]: What about [`ramdisk_image`/`ramdisk_size`](https://github.com/qemu/qemu/blob/master/hw/i386/x86-common.c#L879-L880)?

File: content/blog/2024-07-10-unikraft-gsoc-lxboot_x86.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## Integration with Unikraft
+
+Although debugging could be done only by running the `mklinux_x86.py` script and looking at the header, I wanted to test by integrating everything in Unikraft.
+For doing this I had to modify the following files: [Linker.uk](https://github.com/unikraft/unikraft/blob/staging/plat/kvm/Linker.uk), [Config.uk](https://github.com/unikraft/unikraft/blob/staging/plat/kvm/Config.uk) and [Makefile.rules](https://github.com/unikraft/unikraft/blob/staging/plat/common/Makefile.rules).
+
+## Next steps
+
+At the moment, when I try to run, it loops indefinitely.
+I have to debugg and find out what does not work.
```
Comment (mogasergiu) [inline content/blog/2024-07-10-unikraft-gsoc-lxboot_x86.mdx:80]:
```suggestion
I have to debug and find out what does not work.
```

**Review by mogasergiu — CHANGES_REQUESTED** [review]
Looks good.
Following our discussion, you can also specify how next you are going to look into SeaBIOS/qboot/GRUB2 to see how the jumping to the kernel happens and write the needed 16-bit and 32-bit assembly stubs.

File: content/blog/2024-07-10-unikraft-gsoc-lxboot_x86.mdx (hunk 1 above)
Comment (Mihnea0Firoiu) [reply content/blog/2024-07-10-unikraft-gsoc-lxboot_x86.mdx:53]: I just made the change, I don't know how I forgot to add them.

**Review by StefanJum — APPROVED** [review]
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**